omod-1.10/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_10/OrderResource1_10.java

```diff
@@ -102,6 +108,43 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		}
 	}
 	
+	@Override
+	public Model getCREATEModel(Representation rep) {
+		ModelImpl model = new ModelImpl()
+		        .property("encounter", new StringProperty().example("uuid"))
+		        .property("action", new StringProperty()
+		                ._enum(SwaggerSpecificationCreator.getEnumsAsList(Order.Action.class)))
+		        .property("accessionNumber", new StringProperty())
+		        .property("dateActivated", new DateProperty())
+		        .property("scheduledDate", new DateProperty())
+		        .property("patient", new StringProperty().example("uuid"))
```

Comment:
http://localhost:8080/openmrs/module/webservices/rest/apiDocs.htm#!/order/createOrder
See in the 'Example Value' section.

{
  "encounter": "uuid",
  "action": "NEW",
  "accessionNumber": "string",
  "dateActivated": "2017-08-24",
  "patient": "**uuid**",
  "concept": "uuid",
...
}